scripts/dmaketgz

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+# docker-maketgz
+#
+#***************************************************************************
+#                                  _   _ ____  _
+#  Project                     ___| | | |  _ \| |
+#                             / __| | | | |_) | |
+#                            | (__| |_| |  _ <| |___
+#                             \___|\___/|_| \_\_____|
+#
+# Copyright (C) Daniel Stenberg, <daniel@haxx.se>, et al.
+#
+# This software is licensed as described in the file COPYING, which
+# you should have received as part of this distribution. The terms
+# are also available at https://curl.se/docs/copyright.html.
+#
+# You may opt to use, copy, modify, merge, publish, distribute and/or sell
+# copies of the Software, and permit persons to whom the Software is
+# furnished to do so, under the terms of the COPYING file.
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+# KIND, either express or implied.
+#
+# SPDX-License-Identifier: curl
+#
+###########################################################################
+
+version="${1:-}"
+
+if [ -z "$version" ]; then
+  echo "Specify a version number!"
+  exit
+fi
+
+make distclean
+docker build \
+       --build-arg SOURCE_DATE_EPOCH="$(date -u +%s)" \
+       --build-arg UID="$(id -u)" \
+       --build-arg GID="$(id -g)" \
+       -t curl/curl .
+
+run="run --rm -it -u $(id -u):$(id -g) -v $(pwd):/usr/src -w /usr/src curl/curl"
```

Comment:
Ow, I believe now that we are creating a user with a home dir, we need to run this in the home dir, too

https://github.com/curl/curl/blob/41c03b4c98dbc639a32d32486ed5146be2e73ee1/Dockerfile#L33-L36

as in

```bash
docker run --rm -it -u $(id -u):$(id -g) \
  -v $(pwd):/home/dev/curl -w /home/dev/curl \
  curl/curl bash
```

this is what I tested locally and should get you files where the host:container user:group are the same.

Reply:
Does it really matter where it runs? The generated output tarballs seem fine to me...

Reply:
Interesting! I expected the `dev` user not having permissions to `/usr/src/` but if it works for you it works for you!